generators/app/templates/_dockerTaskGeneric.sh

```diff
@@ -26,10 +26,18 @@ buildImage () {
       taggedImageName="$imageName"
       if [[ $ENVIRONMENT != "release" ]]; then
         taggedImageName="$imageName:$ENVIRONMENT"
-      fi
+      fi<% if (projectType === 'aspnet' && dotnetVersion === 'RC2') { %>
+
+      echo "Building the project ($ENVIRONMENT)."
+      pubFolder="bin/$ENVIRONMENT/netcoreapp1.0/publish"
+      dotnet build -c $ENVIRONMENT
+      dotnet publish -c $ENVIRONMENT -o $pubFolder
```

Comment:
Apparently not as it worked ;) But I agree that would be better form.

---
In reply to: [67467303](https://github.com/Microsoft/generator-docker/pull/69#discussion_r67467303) [](ancestors = 67467303)